docs/operator-guide/upgrade-edp-3.9.md

```diff
@@ -0,0 +1,90 @@
+# Upgrade EDP v3.8 to 3.9
+
+!!! Important
+    We suggest backing up the EDP environment before starting the upgrade procedure.
+
+This section provides detailed instructions for upgrading the EPAM Delivery Platform to version 3.9.0. Follow the steps and requirements outlined below:
+
+!!! Warning
+    EDP-Install version v.3.9 is not support Kiosk as tenancy engine. Please migrate to Capsule engine or disable this options.
```

Comment:
Since the version 3.9.0, KubeRocketCI no longer supports Kiosk as a tenancy engine. Please migrate to the Capsule engine or disable this option.